apps/i18n/studio/en_us.json

```diff
@@ -45,6 +45,7 @@
   "catVariables": "Variables",
   "changeScoreTooltip" : "Add or remove a point to the score.",
   "changeScoreTooltipK1" : "Add a point to the score.",
+  "clickToReset" : "Click to reset",
```

Comment:
We should consider text that is mobile friendly as well, ("click" vs "tap") - not sure what is the best solution.